ts/components/PspComponent.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
```

Comment:
What do you think to about moving this component in `ts/components/wallet/payment/PspComponent.tsx`